datalake/src/test/java/com/sequenceiq/datalake/service/sdx/SdxExternalDatabaseConfigurerTest.java

```diff
@@ -23,6 +24,9 @@ public class SdxExternalDatabaseConfigurerTest {
     @Mock
     private PlatformConfig platformConfig;
 
+    @Mock
+    private DatabaseDefaultVersionProvider databaseDefaultVersionProvider;
```

Comment:
is there any interraction on this mock ?

Reply:
yes, thats why I have added

Reply:
there is not assertion on it. Can you please add then ?

Reply:
sure thing, added